src/routes/OrganizationsList/OrganizationsList.tsx

```diff
@@ -26,6 +26,8 @@ import {Suspense, useEffect, useState} from 'react';
 import {
   bulkDeleteOrganizations,
   fetchAllOrgs,
+  fetchOrgsAsSuperUser,
+  SuperUserOrganizations,
```

Comment:
can we normalize the response from the API in the Resource file instead? We should only be exposing `IOgranization` ideally

Reply:
Sounds good, edited to return an organization list: `IOgranization[]`